testing/tests/DevExpress.ui.widgets.editors/textEditorParts/common.tests.js

```diff
@@ -477,17 +478,31 @@ QUnit.module('label integration', {
                 .dxTextEditor('instance');
             this.$input = this.$textEditor.find(`.${INPUT_CLASS}`);
         };
+
+        class TextEditorLabelMock extends TextEditorLabel {
+            updateMaxWidth = sinon.stub()
+            updateBeforeWidth = sinon.stub()
+            updateMode = sinon.stub()
+            updateText = sinon.stub()
+            updateMark = sinon.stub()
+            updateContainsButtonsBefore = sinon.stub()
+        }
+
+        this.TextEditorLabelMock = (args) => { this.labelArgs = args; return this.labelMock = new TextEditorLabelMock(args); };
+
+        TextEditor.mockTextEditorLabel(this.TextEditorLabelMock);
+    },
+    afterEach: function() {
+        Object.values(this.labelMock, (stub) => {
```

Comment:
We can create a sinon [SandBox](https://sinonjs.org/releases/latest/sandbox/) in beforeEach to simplify the clearing